index.html

```diff
@@ -652,24 +658,50 @@ <h3>
           fails, set both <a data-link-for=
           "PerformanceNavigationTiming">unloadEventStart</a> and
           <a data-link-for="PerformanceNavigationTiming">unloadEventEnd</a> to
-          0 then go to <a href="#fetch-start-step">fetch-start-step</a>.
+          0 then go to <a href="#worker-start-step">worker-start-step</a>.
           Otherwise, record <a data-link-for=
           "PerformanceNavigationTiming">unloadEventStart</a> as the time
           immediately before the unload event.
           </li>
           <li>Immediately after the unload event is completed, record the
           current time as <a data-link-for=
-          "PerformanceNavigationTiming">unloadEventEnd</a>. If the navigation
-          URL has an <a data-cite="service-workers#dfn-active-worker">active
-          worker registration</a>, immediately before the user agent
-          <a data-cite="service-workers#run-service-worker">runs the worker</a>
-          record the time as {{workerStart}}, or if the worker is available,
-          record the time before the <a data-cite=
-          "service-workers#on-fetch-request-algorithm">event named `fetch` is
-          fired</a> at the active worker. Otherwise, if the navigation URL has
-          no matching <a data-cite=
-          "service-workers#dfn-service-worker-registration">service worker
-          registration</a>, set {{workerStart}} value to zero.
+          "PerformanceNavigationTiming">unloadEventEnd</a>.
+          </li>
+          <li id="worker-start-step">
+            <i>[worker-start-step]</i>
+              <ul>
+                <li>If the [=request/current url=]
+                  has no matching <a data-cite=
+                  "service-workers#dfn-service-worker-registration">service worker
+                  registration</a>, set {{workerStart}} to a {{DOMHighResTimeStamp}}
+                  with a time value of 0, and go to the
```

Comment:
nit: replace "a DOMHighResTimeStamp with a time value of 0" with "0".

Reply:
Thought I had found all these -- fixed!